### PR TITLE
Landfill ej

### PR DESCRIPTION
**Pull request by WeathermanTrent**
## Why are you creating this Pull Request?

- [Adding Datasets or Stories](?title=Content%3A%20%3Cname%3E&expand=1&template=content.md)
- [Version Release](?title=Deploy%20vX.X.X&expand=1&template=version_release.md)
- [Other](?expand=1&template=default.md)

### CODE REVIEW

File: stories/landfill-ej.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,202 @@
+---
+id: 'landfill-ej'
+name: Disproportionate Environmental Burdens - A Study of Landfills
```
Comment (j08lue) [inline stories/landfill-ej.stories.mdx:3]: The front matter is very bare-bones, perhaps lacking the `description` key? Perhaps that is what is causing https://github.com/NASA-IMPACT/veda-ui/issues/916?

**Review by j08lue — CHANGES_REQUESTED** [review]
Please review whether the front matter has all required properties.

**Review by mahineelam — COMMENTED** [review]
edits

File: stories/landfill-ej.stories.mdx (hunk 1 above)
Comment (jerikac) [reply stories/landfill-ej.stories.mdx:3]: @WeathermanTrent Was this addressed? If so, @j08lue this story is ready for publication, pending your/ @hanbyul-here 's review.

**Comment by jerikac** [comment]
@j08lue or @hanbyul-here this story is ready to go live... what needs to happen so it pushes to prod? Can one of you review and merge the PR?
 cc: @mahineelam

File: stories/landfill-ej.stories.mdx (hunk 1 above)
Comment (hanbyul-here) [reply stories/landfill-ej.stories.mdx:3]: This problem doesn't seem to be addressed. Look how the card is missing the description.

Screen Shot 2024-05-13 at 2 56 21 PM

File: stories/landfill-ej.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+media:
+  src: ::file ./landfill.jpg
+  alt: Garbage pile in landfill.
+  author:
+    name: vchal
+    url: https://www.istockphoto.com/photos/landfill
+pubDate: 2024-04-05
```
Comment (hanbyul-here) [inline stories/landfill-ej.stories.mdx:10]: might be better to change this to more recent date?

File: datasets/emit-landfill.data.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+id: landfill-emit
+name: "EMIT Landfill"
+description: "FILL IN"
```
Comment (hanbyul-here) [inline datasets/emit-landfill.data.mdx:4]: Does this mean to be this value?

File: stories/landfill-ej.stories.mdx (hunk 1 above)
Comment (jerikac) [reply stories/landfill-ej.stories.mdx:3]: Let me have @mahineelam draft up a sentence for a description

File: stories/landfill-ej.stories.mdx (hunk 2 above)
Comment (jerikac) [reply stories/landfill-ej.stories.mdx:10]: We can change the pub date to today, or whenever it's actually pushed to prod!

File: stories/landfill-ej.stories.mdx (hunk 1 above)
Comment (mahineelam) [reply stories/landfill-ej.stories.mdx:3]: https://deploy-preview-379--visex.netlify.app/stories/landfill-ej


I remember adding sentence to that. Can you check this one

File: datasets/emit-landfill.data.mdx (hunk 3 above)
Comment (hanbyul-here) [reply datasets/emit-landfill.data.mdx:4]: It shows up in the Catalog view.
Screen Shot 2024-05-13 at 4 23 26 PM

File: stories/landfill-ej.stories.mdx (hunk 1 above)
Comment (jerikac) [reply stories/landfill-ej.stories.mdx:3]: @mahineelam can you provide the language again, or a new one? Looks like the old sentence was deleted or removed on accident.

File: datasets/emit-landfill.data.mdx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        - '#f7cf39'
+
+
+  
+---
+
+<Block>
+  <Prose>
+
+FILL IN
```
Comment (hanbyul-here) [inline datasets/emit-landfill.data.mdx:56]: Oh, how did I miss this before? This should be filled in, too. https://deploy-preview-379--visex.netlify.app/data-catalog/landfill-emit

**Review by dzole0311 — APPROVED** [review]
I went through the changes in the datasets & the stories, and it looks as expected.

**Review by j08lue — APPROVED** [review]
Assuming the author signalled that the content is ready for final review.

Content looks complete and all resources (images and datasets on map) load as I would expect.

File: stories/landfill-ej.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+id: 'landfill-ej'
+name: Disproportionate Environmental Burdens - A Study of Landfills
+description: Examining the environmental, socioeconomic, and health impacts of landfills in Stockton, California and Dallas, Texas.
+media:
+  src: ::file ./landfill.jpg
+  alt: Garbage pile in landfill.
+  author:
+    name: vchal
+    url: https://www.istockphoto.com/photos/landfill
+pubDate: 2024-05-17
```
Comment (j08lue) [inline stories/landfill-ej.stories.mdx:11]:
```suggestion
pubDate: 2024-05-21
```

**Review by j08lue — COMMENTED** [review]
Should update this date before you merge, @dzole0311.

**Pull request merged**